atlasdb-plugin/src/main/groovy/com/palantir/atlasdb/AtlasPluginExtension.groovy

```diff
@@ -0,0 +1,5 @@
+package com.palantir.atlasdb
+
+class AtlasPluginExtension {
```

Comment:
I wonder if we should even be specifying a version number and if the version of the "atlas gradle plugin" should just always be tied to the atlas version itself.